ansible/roles/letsencrypt/templates/project-le-ssl.conf

```diff
@@ -1,28 +1,10 @@
 <IfModule mod_ssl.c>
     <VirtualHost *:443>
-        ServerName {{server_name}}
-        {% for alias in server_aliases %}
-        ServerAlias {{alias}}
-        {% endfor %}
-        RailsEnv {{rails_env}}
-        # !!! Be sure to point DocumentRoot to 'public'!
-        DocumentRoot /u/apps/{{project_name}}/current/public
-        <Directory /u/apps/{{project_name}}/current/public>
-            # This relaxes Apache security settings.
-            AllowOverride all
-            # MultiViews must be turned off.
-            Options -MultiViews
-            # Uncomment this if you're on Apache >= 2.4:
-            Require all granted
-        </Directory>
-        <Location /assets/>
-          # Use of ETag is discouraged when Last-Modified is present
-          Header unset ETag
-          FileETag None
-          # RFC says only cache for 1 year
-          ExpiresActive On
-          ExpiresDefault "access plus 1 year"
-        </Location>
+        {{ apache_config_contents |
+          replace('<VirtualHost *:80>\n    ', '') |
+          replace('</VirtualHost>', '') |
+          indent(4, false)
+        }}
```

Comment:
does this handle redirecting port 80 to 443?  I would almost rather have just a single apache config for the LE enabled ones, I think it will be more straightforward than having the deal with 2 (if we're doing it manually anyway)